### PR TITLE
Fix for w.atwiki.jp

### PR DESCRIPTION
**Pull request by mezeraj**
Fixed the dynamic theme issues that weren't changing colors in dark mode.

### CODE REVIEW

**Comment by mezeraj** [comment]
This is to fix [issue #2717](https://github.com/darkreader/darkreader/issues/2717). I was unsure how to link it or if a moderator did that

**Comment by AndrewDaGuy** [comment]
resolved #2717

**Comment by mezeraj** [comment]
Resolves [#2717](https://github.com/darkreader/darkreader/issues/2717)

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -10176,6 +10176,21 @@ article:hover::before {
...
 ================================
 
+w.atwiki.jp
+
+CSS
+.atwiki-contents-shadow {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.atwiki-jp-bg2{
+    color: var(--darkreader-neutral-background) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10186]: Should the text-color of this element matches the dynamic value of background?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -10176,6 +10176,21 @@ article:hover::before {
 
 ================================
 
+w.atwiki.jp
+
+CSS
+.atwiki-contents-shadow {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10182]:
```suggestion
.atwiki-contents-shadow, 
.main_wrapper {
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -10176,6 +10176,21 @@ article:hover::before {
...
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.atwiki-jp-bg2{
+    color: var(--darkreader-neutral-background) !important;
+}
+.main_wrapper {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10188]: You can remove this entry here, when you've merged the suggestion.

**Review by Gusted — CHANGES_REQUESTED** [review]

**Review by Gusted — APPROVED** [review]

**Pull request merged**